website/www/site/content/en/contribute/release-guide.md

```diff
@@ -1178,6 +1178,10 @@ Create and push a new signed tag for the released version by copying the tag for
 gpg --output ~/doc.sig --sign ~/.bashrc
 
 VERSION_TAG="v${RELEASE_VERSION}"
+RC_TAG="${VERSION_TAG}-RC${RC_NUM}"
+
+# Validate that the variables are non empty. If they're empty.
```

Comment:
```suggestion
# Validate that the variables are non empty.
```

Reply:
I found a more concrete approach with git, and auto-fixing the issue Ritesh ran into with the tag sync.